features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -68,6 +71,11 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
     @Inject
     lateinit var configuration: Configuration
 
+    @Inject
+    lateinit var dataSyncStatus: DataSyncStatus
+
+    val syncProgress by lazy { dataSyncStatus.getSyncProgress() }
```

Comment:
Why not just access it from the service directly? Seems like it will produce the same effect.

Reply:
This was to get around a NPE.

Reply:
Found a way to inject the DataSyncStatus service into the viewModel, so this code was removed.